src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,49 @@
+package tasks;
+
+import java.time.LocalDateTime;
+
+import common.DateTimeParser;
+
+public class Deadline extends Task {
+
+    private String by;
+    private LocalDateTime byDate;
```

Comment:
Would it be better if the variables here are named more descriptively, instead of 'by' and 'byDate'?